packages/yourloops/pages/notifications/notifications.tsx

```diff
@@ -0,0 +1,114 @@
+
+import HeaderBar from "../../components/header-bar";
+import { Roles } from "../../models/shoreline";
+import { useAuth } from "../../lib/auth/hook/use-auth";
+import { INotification, Notification, NotificationType } from "./notification";
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    homeIcon: {
+      marginRight: "0.5em",
+    },
+    breadcrumbLink: {
+      display: "flex",
+    },
+    toolBar: {
+      display: "grid",
+      gridTemplateRows: "auto",
+      gridTemplateColumns: "auto auto auto",
+      paddingLeft: "6em",
+      paddingRight: "6em",
+    },
+  })
+);
+
+const NotificationHeader = () => {
+  const { t } = useTranslation("yourloops");
+  const classes = useStyles();
+  const { user } = useAuth();
+
+  const homePage = useMemo(() => (user?.roles?.length && user.roles[0] === Roles.caregiver ? "/" : "/hcp/patients"), [
+    // FIXME: set the route for caregiver
+    user,
+  ]);
+
+  return (
+    <Fragment>
+      <HeaderBar />
+      <AppBar position="static" color="secondary">
+        <Toolbar className={classes.toolBar}>
+          <Breadcrumbs aria-label={t("breadcrumb")} separator={<NavigateNextIcon fontSize="small" />}>
+            <Link className={classes.breadcrumbLink} color="textPrimary" href={homePage}>
+              <HomeIcon className={classes.homeIcon} />
+              {t("home")}
+            </Link>
+            <div>{t("notifications")}</div>
+          </Breadcrumbs>
+        </Toolbar>
+      </AppBar>
+    </Fragment>
+  );
+};
+
+const sortNotification = (notifA: INotification, notifB: INotification): number =>
+  Date.parse(notifB.date) - Date.parse(notifA.date);
+
+export const NotificationsPage = (): JSX.Element => {
+  const { user } = useAuth();
+
+  const fakeNotif1: INotification = {
+    date: new Date().toISOString(),
+    emitter: { firstName: "Jean", lastName: "Dujardin", role: Roles.clinic },
+    type: NotificationType.joinGroup,
+    target: "Service de Diabétologie CH Angers",
+  };
+  const fakeNotif2: INotification = {
+    date: "2021-02-18T10:00:00",
+    emitter: { firstName: "Jeanne", lastName: "Dubois", role: Roles.patient },
+    type: NotificationType.dataShare,
+  };
+  const fakeNotif3: INotification = {
+    date: new Date(Date.now() - 24 * 60 * 60 * 1000).toISOString(), // yesterday date
+    emitter: { firstName: "Bob", lastName: "L'Eponge", role: Roles.clinic },
+    type: NotificationType.joinGroup,
+    target: "Crabe croustillant",
+  };
+  const notifs: INotification[] = [fakeNotif1, fakeNotif2, fakeNotif3];
+
+  return (
+    <Fragment>
+      <NotificationHeader />
+      <Container maxWidth="lg" style={{ marginTop: "1em" }}>
+        <List>
+          {notifs.sort(sortNotification).map((notification, index) => (
+            <ListItem key={index} style={{ padding: "8px 0" }} divider={index !== notifs.length - 1}>
+              <Notification notification={notification} userRole={user?.roles && user.roles[0]} />
```

Comment:
i would propose to split the notification props in order to remove the duplication.